pgcontents/checkpoints.py

```diff
@@ -65,7 +65,7 @@ def rename_checkpoint(self, checkpoint_id, old_path, new_path):
         """Rename a checkpoint from old_path to new_path."""
         with self.engine.begin() as db:
             return move_single_remote_checkpoint(
-                db, old_path, new_path, checkpoint_id,
+                db, self.user_id, old_path, new_path, checkpoint_id,
```

Comment:
```rename_checkpoint``` is only called by ```rename_all_checkpoints``` in the base class, but ```PostgresCheckpoints``` overrides ```rename_all_checkpoints``` to rename all the checkpoints in a single SQL statement, instead of renaming them one by one.  I'm going to replace this fix with ```raise NotImplementedError()``` to show that it's not currently expected that something would call it.